src/components/toast/ToastContext.tsx

```diff
@@ -8,18 +8,14 @@ import React, {
 
 import { AnimatePresence } from 'framer-motion';
 
-import { Toast } from './Toast';
+import { Toast, ToastProps } from './Toast';
 
 export const ToastContext = createContext((toast: ReactNode): void => {
   const defaultFunction = (options: ReactNode) => options;
   defaultFunction(toast);
 });
 
-type Props = {
-  children: ReactNode;
-};
-
-export const ToastContextProvider = ({ children }: Props) => {
+export const ToastContextProvider = ({ children, intent }: ToastProps) => {
```

Comment:
The ToastContextProvider exists as a Parent of most of the App. With how this is coded it would mean we would need to pick one toast intent for the entire Application. I'm afraid we need to go back to the drawing board and discuss the architecture of this change.

Reply:
Hmn. Yah you are right, I forgot about this wrapper. I think if we do some tweaks on the way `addToast` function work in the `ToastContextProvider`, and add 1 prop for `intent` into `ToastContext` component, it may work 🤔 . What do you think?

Reply:
What would it look like in the Dashboard when it is consumed?

Reply:
I just imagine how it would work but haven't played with it yet. Correct me if I'm wrong, I think that `notify` in Dashboard is the same as `addToast` function, so if we just add `intent` into both `addToast` and `ToastContext`. We can use `notify('text', 'primary')` in Dashboard 🤔

Reply:
That syntax looks good to me. 👍